docs/pages/reference/helm-reference/teleport-kube-agent.mdx

```diff
@@ -106,6 +106,90 @@ This parameter is not mandatory to preserve backwards compatibility with older c
 
 If you specify a role here, you may also need to specify some other settings which are detailed in this reference.
 
+## `updater`
+
+`updater` controls whether the Kube Agent Updater should be deployed alongside
+the `teleport-kube-agent`. The updater fetches the target version, validates the
+image signature, and updates the teleport deployment.
+
+All kubernetes-specific fields such as `tolerations`, `affinity`, `nodeSelector`,
+... default to the agent values. However, they can be overridden from the
+`udpater` object. For example:
+
+```yaml
+# the agent pod requests 1cpu and 2 GiB of memory. It also has a memory limit.
+resources:
+  requests:
+    cpu: "1"
+    memory: "2Gi"
+  limits:
+    memory: "2Gi"
+
+# the updater pod requests 0.5 cpu and 512MiB of memory. The memory limit has also been unset.
+updater:
+  resources:
+    requests:
+      cpu: "0.5"
+      memory: "512Mi"
+    limits: ~
+```
+
+Other updater-specific values that can be defined in `updater` are described
+below:
+
+### `updater.enabled`
+
+Enables the Kube Agent Updater and deploys it alongside the Teleport Agent.
+You must enable this when:
```

Comment:
Nit: I think "must" is a bit too strong a word here and below. I would rephrase to "updater should be enabled ...".

Reply:
I think the docs folks don't like should (at least I got the feedback multiple times). I can change this from a MUST to a CAN.